server/service/integration_enterprise_test.go

```diff
@@ -8463,6 +8464,7 @@ func (s *integrationEnterpriseTestSuite) TestAllSoftwareTitles() {
 	// valid title
 	stResp = getSoftwareTitleResponse{}
 	s.DoJSON("GET", fmt.Sprintf("/api/latest/fleet/software/titles/%d", fooTitle.ID), getSoftwareTitleRequest{}, http.StatusOK, &stResp)
+	s.NotNil(*stResp.SoftwareTitle.CountsUpdatedAt)
```

Comment:
could be useful to also assert that the timestamp is not the zero value of time.Time.  maybe something like `time.Now().Add( -time.Hour)`

Reply:
Used the nonzero assertion in [bd62a40](https://github.com/fleetdm/fleet/pull/23478/commits/bd62a40d2c65757ecf314e5407eaed783896733c). That work?